defs/scenarios/juju/charms_bugs.yaml

```diff
@@ -3,6 +3,12 @@ checks:
     input:
       path: 'var/log/juju/unit-ceph-osd*.log'
     expr: '.* Cannot zap a device used by lvm'
+
```

Comment:
can we lose the newline chars either side of this please?

Reply:
Not sure what you mean by that. Most checks are like this. Do you have an example?